packages/mermaid/src/utils.spec.ts

```diff
@@ -326,7 +326,7 @@ describe('when formatting urls', function () {
     expect(result).toEqual(url);
 
     result = utils.formatUrl(url, { securityLevel: 'strict' });
-    expect(result).toEqual('about:blank');
+    expect(result).toEqual('javascript:alert(%22test%22)');
```

Comment:
If this turns out to be an issue, we can also consider using a different url sanitization library, ill leave that up to the maintainers to decide :)

Reply:
@knsv

Reply:
I am a bit worried that this might cause XSS attacks (although, it might be impossible without using URL unsafe characters like `"`, as they'll get converted to `%22`).

If somebody makes a malicious mermaid diagram with a malicious `javascript:xxxx` link, hosts it on example.com, and convinces somebody to click it, I believe the JavaScript will have access to the user's data on example.com

However, it's possible that https://github.com/cure53/DOMPurify will handle that for us, though, even if it isn't handled by the URI sanitizer library. I'll let @knsv chime in on it, since he probably knows more!